core/src/test/java/org/elasticsearch/search/aggregations/bucket/SignificantTermsSignificanceScoreIT.java

```diff
@@ -273,12 +272,15 @@ public void testXContentResponse() throws Exception {
         }
 
         XContentBuilder responseBuilder = XContentFactory.jsonBuilder();
+        responseBuilder.startObject();
         classes.toXContent(responseBuilder, null);
+        responseBuilder.endObject();
```

Comment:
same question here if Strings.toString(XContentBuilder, true) could be used, and also for https://github.com/elastic/elasticsearch/pull/20011/files#diff-4dad2c5391932f86a402ee44b5160218R57